hw/top_earlgrey/dv/chip_rom_tests.hjson

```diff
@@ -19,7 +19,7 @@
     // Signed chip-level tests to be run with ROM, instead of test ROM.
     {
       name: chip_sw_uart_smoketest_signed
-      uvm_test_seq: chip_sw_base_vseq
+      uvm_test_seq: chip_sw_uart_smoke_vseq
```

Comment:
not sure why we need this vseq. IIUC, uart_smoketest doesn't need a uart agent, since it just does loopback.

Reply:
It needs UART 0 to be connected. By default, NONE of the functional interfaces are connected to the chip IOs. Only the dedicated ones are. This is the main idea of this change - it enforces IO connection / disconnection to extended sequences so that there is no confusion.

Reply:
Maybe I'm wrong. seems like `uart_smoketest` is using loopback mode, so it doesn't need uart0 to be connected. It can be a pure SW test, right?

Reply:
The default weak pulldowns cause the test to fail.

Reply:
I see. That makes sense. RX can't be zero